src/launchWizard/launchWizard.ts

```diff
@@ -52,12 +53,15 @@ async function createUpdateConfigFile(data, updateOrCreate) {
     fs.mkdirSync(`${rootPath}/.vscode`)
   }
 
+  const launchPath =
+    os.platform() === 'win32'
```

Comment:
I can hardly believe you'll get away without needing something like this:
```
osPath(`${rootPath}/.vscode/launch.json`)
```
Which buries the os-specific check down where it is not clutter. 

We made this mistake of allowing OS conditionalizations to proliferate in the CLI test code in Daffodil, and since we didn't nip it in the bud, there's an open ticket to basically convert hundreds of tests that have inline OS conditionalizations to use an osPath like library and otherwise push this os/path stuff down a level.

Reply:
So from my testing on a Windows VM, I believe we don't need the osPath. I believe the open text document file function may be converting paths under the hood as I had never had issues with the class paths until trying to load the style and script files for the launch wizard. Everything else worked fine I assume because it may be handled by a lot of the VSCode API functions by default

Reply:
Ok. I'm fine with that if the basic libraries already are dealing with this. 

 It seems odd that this is the only place we need to do OS conditionalization though.

Reply:
If the basic libraries are really covering this, and this is likely one of the only places we have to do os conditionalization (on paths anyway) then I think this code needs a comment to that effect. 

Reading this code, a naive typescript coder (like me in a year when I've forgotten all this) would just start proliferating these os.platform() === 'win32' checks. We want to avoid that.

Reply:
Yeah so from my understanding most things handle this but in this case we are constructing the entire path to our file based on the folder and on Windows it needs an additional `/` at the beginning of the path.